K8S.md

```diff
@@ -0,0 +1,48 @@
+# [kubevirtci](README.md): Getting Started with a multi-node Kubernetes Provider                              
+                                                                                      
+Download this repo                                                                    
+```                                                                                   
+git clone https://github.com/kubevirt/kubevirtci.git                                  
+cd kubevirtci                                                                         
+```                                                                                   
+                                                                                      
+Start multi node k8s cluster with 2 nics                                              
+```                                                                                   
+export KUBEVIRT_PROVIDER=k8s-1.13.3 KUBEVIRT_NUM_NODES=2 KUBEVIRT_NUM_SECONDARY_NICS=1
```

Comment:
Lets set the default provider to be 1.17 or 1.16

Reply:
Will set it to 1.16.2.